src/panels/config/zwave/zwave-network.ts

```diff
@@ -1,52 +1,49 @@
 import "@polymer/paper-icon-button/paper-icon-button";
-import { html } from "@polymer/polymer/lib/utils/html-tag";
-import { PolymerElement } from "@polymer/polymer/polymer-element";
+
+import {
+  css,
+  CSSResult,
+  html,
+  LitElement,
+  PropertyDeclarations,
+  TemplateResult,
+} from "lit-element";
+
+import { haStyle } from "../../../resources/styles";
+import { HomeAssistant } from "../../../types";
 
 import "../../../components/buttons/ha-call-api-button";
 import "../../../components/buttons/ha-call-service-button";
 import "../../../components/ha-service-description";
 import "../../../components/ha-card";
 import "../ha-config-section";
 
-class ZwaveNetwork extends PolymerElement {
-  static get template() {
-    return html`
-      <style include="iron-flex ha-style">
-        .content {
-          margin-top: 24px;
-        }
-
-        ha-card {
-          margin: 0 auto;
-          max-width: 600px;
-        }
-
-        .card-actions.warning ha-call-service-button {
-          color: var(--google-red-500);
-        }
+export class ZwaveNetwork extends LitElement {
+  public hass?: HomeAssistant;
+  public isWide?: boolean;
+  private _showHelp: boolean;
```

Comment:
Instead of `static get properties`, just annotate the properties with `@property`. And to add default values, just set it inline, like this:

```suggestion
  @property() private _showHelp = false;
```